great-idea/index.html

```diff
@@ -16,7 +16,66 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <div class="container">
+      
+           <div class="navigation">
+               <nav>
+                  <a href="#">Services</a>
+                  <a href="#">Product</a>
+                  <a href="#">Vision</a>
+                  <a href="#">Features</a>
+                  <a href="#">About</a>
+                  <a href="#">Contact</a>
+               </nav>
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+          </div>
+      <header> 
+        <h1> Innovation<br> On<br> Demand</h1>
+          <div class="button">
+            <button>Get Started</button>
+          </div>
+          <img class="header-picture" src="img/header-img.png" alt="Image of a code snippet.">
+      </header>
+      
+  <section class="middle-content">
+    <div class="top-middle-content">
+      <h2>Features</h2>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+  <div class="top-middle-content">
+    <h2>About</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+  </section>
+  <div class="middle-img">
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+</div>
+  <section class="bottom-content">
```

Comment:
you want to use more spacing for separating different sections and container divs it allows to read the code much easier.